tests/cases.py

```diff
@@ -1670,10 +1670,10 @@ def _get_input(
             triples = self.factory.mapped_triples.to(self.model.device)
             if target == LABEL_HEAD:
                 sel_col, start_col = 0, 1
+            elif target == LABEL_RELATION:
+                sel_col, start_col = 1, 2
             elif target == LABEL_TAIL:
                 sel_col, start_col = 2, 0
-            elif target == LABEL_RELATION:
-                raise NotImplementedError(target)
             else:
                 raise ValueError(target)
             stop_col = start_col + 2
```

Comment:
do we need to adjust this one, too?

Reply:
is this one resolved? from the code below it looks like the range `start_col:stop_col` is not working for all but the first case